src/main/drivers/light_ws2811strip.c

```diff
@@ -37,6 +37,7 @@
 #include "common/color.h"
 #include "common/colorconversion.h"
 #include "dma.h"
+#include "io.h"
```

Comment:
> we are not including megabytes of header files ...

Well, actually, we are. At current count there are 50 targets and the total size of the headers is about 1.3 megabytes. So for a full build we include over 50 megabytes of headers. The travis build is a partial build of only 15 targets and that can take 10 minutes elapsed time, and over 30 minutes total time.

**We have a significant build time problem**. I agree that this minor optimisation does not achieve significant benefit on its own, but as I said the accumulation of many small optimisations like this is significant.